generate-stackbrew-library.sh

```diff
@@ -12,11 +12,12 @@ declare -A debianSuites=(
 	[2.3]='jessie'
 	[2.4]='jessie'
 )
-defaultAlpineVersion='3.6'
+defaultAlpineVersion='3.7'
 declare -A alpineVersions=(
 	[2.2]='3.4'
 	[2.3]='3.4'
 	[2.4]='3.4'
+	[2.4]='3.6'
```

Comment:
Doh, sorry, I didn't realize this already had an entry for 2.4, I told you wrong. 😞